tests/jsTests/menuSpec.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Suite : Tests for showing and hiding of menu
+ * Description : This is a test suite for showing and hiding menu for smaller screens.
```

Comment:
@fatimarafiqui  Does it mean that these methods are not being used larger screens? Why just smaller screens? 

It would be nice if the comment indicates what are you actually testing, for instance: 
1. This suite tests methods showMenu() and hideMenu() to see if they are being called when <this> action is performed
2. You can mention which file you are actually writing unit tests for

When I read the comment, particularly "max-width 932px", I get these questions:
1. Why just smaller screens and why just max-width 932px
2. Does it mean these methods would not be used for screens of other sizes
3. Does that mean the code does not accomodate other screen sizes? eg: Mobile Web.